tests/unit/TheLow.t.sol

```diff
@@ -174,4 +174,20 @@ contract TheLowTest is Test {
         low.divideRoundUp(100,0,1);
     }
 
+        function testRoyalty() public {
+            vm.startPrank(team);
+            low.transferFrom(team,alice,2);
```

Comment:
Not blocking and we don't even have CI pipeline setup, but `forge fmt` should be run before opening a PR and for instance that would re-format this line's formatting.